src/ose.js

```diff
@@ -2,32 +2,32 @@
  * @file The entry point for the OSE system
  *       We should handle most of our setup here.
  */
-import { OseActorSheetCharacter } from "./module/actor/character-sheet";
+import OseActorSheetCharacter from "./module/actor/character-sheet";
 import OseDataModelCharacter from "./module/actor/data-model-character";
 import OseDataModelMonster from "./module/actor/data-model-monster";
-import { OseActor } from "./module/actor/entity";
-import { OseActorSheetMonster } from "./module/actor/monster-sheet";
+import OseActor from "./module/actor/entity";
+import OseActorSheetMonster from "./module/actor/monster-sheet";
 
 import OseDataModelAbility from "./module/item/data-model-ability";
 import OseDataModelArmor from "./module/item/data-model-armor";
 import OseDataModelContainer from "./module/item/data-model-container";
 import OseDataModelItem from "./module/item/data-model-item";
 import OseDataModelSpell from "./module/item/data-model-spell";
 import OseDataModelWeapon from "./module/item/data-model-weapon";
-import { OseItem } from "./module/item/entity";
-import { OseItemSheet } from "./module/item/item-sheet";
+import OseItem from "./module/item/entity";
+import OseItemSheet from "./module/item/item-sheet";
 
 import * as chat from "./module/chat";
-import { OseCombat } from "./module/combat";
-import { OSE } from "./module/config";
-import { registerFVTTModuleAPIs } from "./module/fvttModuleAPIs";
-import { registerHelpers } from "./module/helpers";
+import OseCombat from "./module/combat";
+import OSE from "./module/config";
+import registerFVTTModuleAPIs from "./module/fvttModuleAPIs";
+import handlebarsHelpers from "./module/helpers";
 import * as macros from "./module/macros";
 import * as party from "./module/party";
-import { OsePartySheet } from "./module/party/party-sheet";
-import { preloadHandlebarsTemplates } from "./module/preloadTemplates";
+import OsePartySheet from "./module/party/party-sheet";
+import templates from "./module/preloadTemplates";
```

Comment:
is this name misleading? or should the filename be changed?

Reply:
I don't find it misleading, though I think we could rename the file to `preloadedTemplates` instead.

We should tackle this when we move our templates from HTML to HBS.